modulefiles/build.container.intel.lua

```diff
@@ -2,15 +2,17 @@ help([[
 Load environment to compile UFS_UTILS in a container using Intel
 ]])
 
-prepend_path("MODULEPATH", "/opt/spack-stack/spack-stack-1.8.0/envs/unified-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/opt/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
 
 stack_intel_ver=os.getenv("stack_intel_ver") or "2021.10.0"
-load(pathJoin("stack-intel", stack_intel_ver))
+stack_impi_ver=os.getenv("stack_impi_ver") or "2021.9.0"
 
-stack_impi_ver=os.getenv("stack_impi_ver") or "2021.12.1"
+load("gnu")
+load(pathJoin("stack-intel", stack_intel_ver))
 load(pathJoin("stack-intel-oneapi-mpi", stack_impi_ver))
+unload("gnu")
```

Comment:
Sorry I didn't get to this earlier. I see its merged, but wanted to know why we decided to load then unloading gnu?